lib/krane/cluster_resource_discovery.rb

```diff
@@ -101,6 +99,22 @@ def version_for_kind(versions, kind)
       version_override.fetch(kind, latest)
     end
 
+    def get_resource_api_path(api_versions, resource)
+      apiversion = resource['apiversion'].to_s
+      apiversion = 'core/v1' if apiversion == 'v1'
+
+      ## In kubectl 1.20 APIGroups was replaced by APIVersions
+      if apiversion.empty?
+        apigroup = resource['apigroup'].to_s
+        group_versions = api_versions[apigroup]
+
+        version = version_for_kind(group_versions, resource['kind'])
+        apigroup = 'core' if apigroup.empty? && version == 'v1'
```

Comment:
Loosely related to [comment above](https://github.com/Shopify/krane/pull/777/files#r552277061), do we strictly require `&& version == "v1"` to determine `apigroup` is core? Is the absence of `apigroup` not enough?

Reply:
Based on the above comment I removed the version check and basing it on the api group missing :)